docs/slimmer_templates.md

```diff
@@ -10,6 +10,12 @@ This is the same as the `gem_layout` template, except that this layout **doesn't
 
 Use this layout if you want to have full width content - such as the blue welcome bar on the GOV.UK homepage.
 
+## `gem_layout_account`
+
+This is a variation on the `gem_layout` template intended to be used on GOV.UK Account pages. This layout **doesn't** include the feedback component as the account pages use a different one from the rest of GOV.UK.
```

Comment:
You've probably already had this conversation, but I wonder if now the accounts stuff is going to be part of `frontend`, why not use the existing feedback mechanism instead of having a whole separate one just for accounts?

Reply:
Yes you're completely right: for now we're trying to have this new account home page in `frontend` match the manage/security section layout as closely as possible, but in the longer term we definitely need to look at integrating account feedback into the existing feedback system.